pkg/gardenlet/controller/backupbucket/reconciler.go

```diff
@@ -69,8 +69,13 @@ type Reconciler struct {
 func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	log := logf.FromContext(ctx)
 
+	gardenCtx, cancel := context.WithTimeout(ctx, controllerutils.DefaultReconciliationTimeout)
+	defer cancel()
+	seedCtx, cancel := context.WithTimeout(ctx, controllerutils.DefaultReconciliationTimeout/2)
```

Comment:
`30s` is a pretty tight timeout for the extension controller.
Let's use a different value than `DefaultReconciliationTimeout` in this controller.
Proposal: `4m` for `gardenCtx` and `3m` for `seedCtx`?

Reply:
I realized just now, that the controller doesn’t actually block and doesn’t wait for the extension object to be ready. With this, we don’t need the high timeout and can keep the default one.